openapi3filter/validate_request.go

```diff
@@ -139,9 +137,12 @@ func ValidateRequestBody(c context.Context, input *RequestValidationInput, reque
 
 	inputMIME := req.Header.Get("Content-Type")
 	mediaType := parseMediaType(inputMIME)
-	if !isMediaTypeJSON(mediaType) {
-		// We support validation of JSON data only.
-		return nil
+	if mediaType == "" {
+		return &RequestError{
+			Input:       input,
+			RequestBody: requestBody,
+			Reason:      "a content type of the request's body is missed",
```

Comment:
`"content type of request body is missing"`

Reply:
Done.